documentation/user_docs/docs/manual/Planning_a_Horace_scan.rst

```diff
@@ -2,6 +2,12 @@
 Planning a Horace scan
 ######################
 
+.. warning::
+
+   While Horace includes its own planner it is potentially less feature complete
+   than the [MANTID DGS
+   planner](https://docs.mantidproject.org/v3.8.0/interfaces/DGSPlanner.html)
```

Comment:
can we say something about when to use which of these two? Otherwise it's just frustrating

Reply:
They're basically the same, there are only two features which Mantid has extra:

* It allows the crystal orientation to be specified as a UB matrix instead of as the `u` and `v` vectors which define the horizontal scattering plane
* It allows rotations about horizontal rotation axes (Horace only allows rotation about the vertical axis).

These are really minor features - Horace does not allow computing `sqw` files generated by rotation about either horizontal axes (parallel to or perpendicular to the incident beam), so it doesn't make sense to support the second feature (Mantid `MDWorkspace`s can be generated from scans about the horizontal axes).

I think actually how the GUI is presented is much more important:

* The Mantid DGS has a single plot window with a single projection of the 3D q-space into 2D. Horace has 3 plots showing all three projections in to the possible combinations of axes.
* The Mantid DGS GUI is resizeable so is actually viewable on small laptop screens. The Horace GUI is not resizeable so is very difficult to view in small laptop screens.

So, personally I would say - if you have a laptop with a small screen use Mantid, if not use Horace.